source/wp-content/themes/wporg-developer-2023/src/search-results-context/index.php

```diff
@@ -22,8 +22,13 @@ function render( $attributes ) {
 		return '';
 	}
 
-	$posts_per_page = get_query_var( 'posts_per_page' );
 	$results_count = $wp_query->found_posts;
+
+	if ( 0 <= $results_count ) {
```

Comment:
I think this should be changed otherwise the count will not be shown for any count over 0
```suggestion
	if ( 0 = $results_count ) {
```

Reply:
ah... it was `$results_count <= 0` in the first place, and then yoda standard came in..
Thanks.